tasks/add_new_openwrap_partner.py

```diff
@@ -374,13 +384,55 @@ def get_dfp_targeting(self):
             if self.bid_price: 
                 top_set['children'].append(self.bid_price)
 
+            # For HVLI, set both pwtbb and pwtecp with OR condition
+            # Targeting keys will be formed like
+            # pwtecp IS (1.*) AND pwtbst=1 AND pwtpid = abc
+            # or
+            # pwtpb = 1.0 AND pwtbst=1 AND pwtpid = abc
+            if self.price_bucket:
+                #pwtecp
+                cust_key_set1 = {
+                    'xsi_type': 'CustomCriteriaSet',
+                    'logicalOperator': 'AND',
+                    'children': []
+                }
+                cust_key_set1['children'].append(top_set)
+                cust_key_set1['children'].append(self.price_set)
+
+                #pwtpb
+                cust_key_set2 = {
+                    'xsi_type': 'CustomCriteriaSet',
+                    'logicalOperator': 'AND',
+                    'children': []
+                }
+                cust_key_set2['children'].append(top_set)
+                cust_key_set2['children'].append(self.price_bucket)
+
+                #final keys will be OR of both sets
+                final_cust_keys = {
+                    'xsi_type': 'CustomCriteriaSet',
+                    'logicalOperator': 'OR',
+                    'children': [cust_key_set1, cust_key_set2 ]
+                }
+                return final_cust_keys
```

Comment:
Currently we are setting final_cust_keys as   `(top_set  AND price_set ) OR (top_set AND price_bucket)` where top_set would have other targeting keys. 

Can something like below code work, basically ANDing of  `price_set OR price_bucket` with other keys?
```
cust_set =   price_set OR price_bucket
top_set['children'].append(cust_set)
```

Reply:
Initially, I tried this approach, but GAM doesn't allow and throws an API error. Even the same is not allowed from the UI also. So that's why I opted for this route. 
I have added a comment on the [JIRA](https://inside.pubmatic.com:9443/jira/browse/UOE-10303) for the same.